h2o-py/tests/pyunit_math/pyunit_ks_metric.py

```diff
@@ -19,6 +20,21 @@ def kolmogorov_smirnov():
     print(ks_verification)
     assert round(ks, 5) != round(ks_verification, 5)
 
+    # Test with specific thresholds
+    model = H2OGradientBoostingEstimator(ntrees=1, gainslift_bins=5)
+    model.train(x=["Origin", "Distance"], y="IsDepDelayed", training_frame=airlines)
+    ks = model.kolmogorov_smirnov(thresholds=[0.01, 0.5, 0.99])
```

Comment:
Have you tried to run the test? This is not working even if you add the method to binomial.py. The goal is not change and test `model.kolmogorov_smirnov()` but add and test `model.model_performance(data).kolmogorov_smirnov()`

```suggestion
```